internal/service/bypass.go

```diff
@@ -0,0 +1,42 @@
+package service
+
+import (
+	"context"
+	"io"
+)
+
+type bypassCache string
+
+// BypassKey key is used to bypass the operation of the service.
+var BypassKey bypassCache = "bypassCacheKey" // nolint: gochecknoglobals
+
+type bypassService interface {
+	Get(ctx context.Context, key string) (io.ReadCloser, error)
+	Put(ctx context.Context, key string, payload io.Reader) error
+}
+
+// Bypass is used to bypass the operations of this struct.
+type Bypass struct {
+	Service bypassService
+}
+
+// Get object.
+func (b Bypass) Get(ctx context.Context, key string) (_ io.ReadCloser, err error) {
```

Comment:
nit: I would use a factory method that will return a NOPCache if the bypass is `true`

Reply:
What do you mean by return a `NOPCache`? Like an error?
I implemented it like this because at the moment we're chaining implementations on that interface (get and put). We have the cache and cipher and now I'm introducing the third element, the bypass.

About the git commit message, I did update it to explain a little more what's happening.